app/editor/src/features/content/form/ContentForm.tsx

```diff
@@ -179,207 +144,169 @@ export const ContentForm: React.FC<IContentFormProps> = ({
     }
   }, [id, fetchContent]);
 
-  const { setShowValidationToast } = useTabValidationToasts();
-
-  const handleSubmit = async (values: IContentForm): Promise<IContentForm> => {
-    let contentResult: IContentModel | null = null;
-    const originalId = values.id;
-    let result = form;
-    try {
-      if (!values.id) {
-        // Only new content is initialized.
-        values.contentType = contentType;
-        values.ownerId = userId;
-      }
+  const handleSubmit = React.useCallback(
```

Comment:
Based on some reading I wanted to see if performance improved by using `useCallback(...)`.